src/bidmanager.js

```diff
@@ -108,11 +108,11 @@ exports.addBidResponse = function (adUnitCode, bid) {
     }
 
     if (!adUnitCode) {
-      utils.logWarn(errorMessage('No adUnitCode was supplied to addBidResponse.'));
```

Comment:
fwiw if you put the bid check before the adUnitCode, you could still use errorMessage here